api/net/Keycloak/CssHelper.cs

```diff
@@ -200,7 +198,7 @@ private async Task AddOrUpdateUserAsync(string username, Entities.User? user, TN
     public async Task<string[]> UpdateUserRolesAsync(string username, string[] roles)
     {
         var userRoles = await _cssService.GetRolesForUserAsync(username);
-        if (userRoles.Users.Length == 0) throw new InvalidOperationException($"User '{username}' does not exist");
+        if (userRoles.Users.Length == 0) return Array.Empty<string>();
```

Comment:
Instead of a failure we just ignore the fact Keycloak Gold Realm did not return the user.